Ву Евгения, HW04

J3ne4ek opened the pull request.

J3ne4ek committed:
first

J3ne4ek committed:
second

trilis commented:
1. ok
2. Не проверяется, что у автомата ровно одна стартовая вершина -- автомат с несколькими стартовыми, очевидно, некорректен

J3ne4ek committed:
add checks

Добавила проверку на множество стартовых вершин, а также новый тест, проверяющий, что ошибка обрабатывается

J3ne4ek commented:
Добавила проверку на множество стартовых вершин, а также новый тест, проверяющий, что ошибка обрабатывается

trilis commented:
ok
